Rocket.Chat/Controllers/Auth/AuthViewControllerGoogleExtensions.swift

```diff
@@ -6,12 +6,13 @@
 //  Copyright © 2017 Rocket.Chat. All rights reserved.
 //
 
+import GoogleSignIn
+
 extension AuthViewController {
     func authenticateWithGoogle() {
         startLoading()
 
-        var configureError: NSError?
-        GGLContext.sharedInstance().configureWithError(&configureError)
+        GIDSignIn.sharedInstance().clientID = "662030055877-9qc3lqsbgif4k1ktl89dqeaec3m163i4.apps.googleusercontent.com"
```

Comment:
I don't like this change but seems like with newer library we have to set it manually instead of calling any `configure` method like it was before.

Reply:
🙈

Reply:
What about the `GoogleService-Info.plist` file?

Reply:
According to documentation https://developers.google.com/identity/sign-in/ios/start-integrating it should be included.

Reply:
But if we're using the file, we can keep using the configureWithError(), like here: https://developers.google.com/identity/sign-in/ios/sign-in?ver=swift.

Reply:
There is no GGLContext class anymore

Reply:
I have removed it and did some tests, seems like it's not needed. Logging with Google account works without this file.